### ISSUE TITLE
PHP script not working

### ISSUE THREAD

**Issue by BobbyOfWindsor1974** [fuel9/DisplayMonkey#78]
Hi, I have PHP installed in the server and if I load the page from the DisplayMonkey folder it works great.  If I let DisplayMonkey show the page in the main display, the script shows as text.  How can I tell DM to parse/run the php script?

**Comment by michaelmalmgren** [comment]
Hi Bobby, I'm not sure I understand what is going on here. Are you referencing a URL in a DM HTML iframe that contains a PHP-script?

**Comment by BobbyOfWindsor1974** [comment]
Yes the frame.

[cid:image001.png@01D46C2E.CD15CB40]

[cid:image002.png@01D46C2E.CD15CB40]



PHP script embeded in the html does not work here.

Thanks

--Bobby


Robert Hicks | Senior Technical Analyst, Schukra - Leggett & Platt Automotive Group- North America
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3 | •: 519-727-7000 Ext. 1003 | :  519-817-1576 |  7: 519-727-7068| •: bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Website: www.leggett-automotive.com [Description: Description: facebook tiny] <https://www.facebook.com/LeggettAuto>   [Description: Description: insta tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: linked in tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: twitter tiny] <https://www.twitter.com/leggettauto/>
P Please consider your environmental responsibility before printing this e-mail

**Comment by michaelmalmgren** [comment]
Bobby, try instead to link in your PHP page as an HTML IFRAME inside the DM HTML frame. I.e. leave your PHP page on a PHP server and link something like this <iframe src="https://myphp_server/myphp_page.php"></iframe>

**Comment by BobbyOfWindsor1974** [comment]
Thanks Michael. I’ll give it a try and let you know. Your help is appreciated.



Thanks

--Bobby


Robert Hicks | Senior Technical Analyst, Schukra - Leggett & Platt Automotive Group- North America
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3 | •: 519-727-7000 Ext. 1003 | :  519-817-1576 |  7: 519-727-7068| •: bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Website: www.leggett-automotive.com [Description: Description: facebook tiny] <https://www.facebook.com/LeggettAuto>   [Description: Description: insta tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: linked in tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: twitter tiny] <https://www.twitter.com/leggettauto/>
P Please consider your environmental responsibility before printing this e-mail

**Comment by psined1** [comment]
I assume your PHP page will be served from a different URL base. This can be a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) challenge in many different browsers...

**Comment by BobbyOfWindsor1974** [comment]
Yes. We have PHP setup on IIS and going directly to the .html file with embedded php, it works.  Only when we add the contents to the HTML container in DM does it now work.



Thanks

--Bobby


Robert Hicks | Senior Technical Analyst, Schukra - Leggett & Platt Automotive Group- North America
360 Silvercreek Industrial Dr., Lakeshore, ON N8N 4Y3 | •: 519-727-7000 Ext. 1003 | :  519-817-1576 |  7: 519-727-7068| •: bobby.hicks@leggett.com<mailto:bobby.hicks@leggett.com>
Website: www.leggett-automotive.com [Description: Description: facebook tiny] <https://www.facebook.com/LeggettAuto>   [Description: Description: insta tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: linked in tiny] <https://www.linkedin.com/company-beta/9362272>   [Description: Description: twitter tiny] <https://www.twitter.com/leggettauto/>
P Please consider your environmental responsibility before printing this e-mail

**Comment by psined1** [comment]
Well, as I said, CORS might be your culprit..

**Comment by BobbyOfWindsor1974** [comment]
We have cors set so we can pull RSS feeds.  It behaves like it doesn’t see the pho script.

Sent from my iPhone

On Oct 25, 2018, at 10:06 AM, Denis P. <notifications@github.com<mailto:notifications@github.com>> wrote:


Well, as I said, CORS might be your culprit..

**Comment by psined1** [comment]
Looks like DM might be downloading your PHP script as a text, rather than executing it on the server. Have you tried HTML frame suggestion from @michaelmalmgren ?

**Comment by BobbyOfWindsor1974** [comment]
I think so as well. It’s not seeing it as php.  The frame with an external HTML/php works well.

Sent from my iPhone

On Nov 12, 2018, at 4:58 PM, Denis P. <notifications@github.com<mailto:notifications@github.com>> wrote:


Looks like DM might be downloading your PHP script as a text, rather than executing it on the server. Have you tried HTML frame suggestion from @michaelmalmgren<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_michaelmalmgren&d=DwMCaQ&c=y2CoG5YnjIJpazuVfWQ7cQ&r=VSK8zsTMB9_o0CyFtzVnI7KU3082pv11Dn5l0G4nfiY&m=lRXoPwEBxLQOo-WlVklutF6zCvLxh1krytx7U3tgpCA&s=YinPXBDrr2l-q-440FHCgj2xX7hNgT-ga3FKzH6dnWQ&e=> ?
